Makefile

```diff
@@ -25,13 +25,14 @@ VERSION := $(shell git describe --tags --dirty --always)
 # strip the first char of the tag if it's a `v`
 VERSION := $(VERSION:v%=%)
 TARBALL_PREFIX := cri-containerd
-TARBALL := $(TARBALL_PREFIX)-$(VERSION).$(GOOS)-$(GOARCH).tar.gz
+TARBALL := $(TARBALL_PREFIX)-$(VERSION).$(GOOS)
```

Comment:
Just see we can keep this as is and change the tar filename based on the GOARCH we supply runtime.